kahuna/public/js/components/gr-collections-panel/gr-collections-panel-node.html

```diff
@@ -0,0 +1,48 @@
+<div ng:init="node = ctrl.node; showChildren = true;">
+    <div class="node__info flex-container" gr:add-to-collection>
+        <div class="node__marker"></div>
+
+        <div class="node__spacer" ng:if="node.data.children.length === 0">
+            <gr-icon></gr-icon>
```

Comment:
It's so we don't have to recreate the styling for the icon to create the exact amount of space it takes up.